package.json

```diff
@@ -4,9 +4,11 @@
   "description": "",
   "main": "index.js",
   "scripts": {
-    "test": "chimp",
-    "test_headless": "HEADLESS=true chimp",
-    "lint": "eslint ./tests/"
+    "test": "CYPRESS_baseUrl=${CYPRESS_baseUrl:=http://localhost:3000/} cypress run",
+    "test_headless": "CYPRESS_baseUrl=${CYPRESS_baseUrl:=http://localhost:3000/} cypress run --browser electron --record --key c9a39109-fae5-476d-b354-21901963024a",
```

Comment:
Sorry, one last comment can we put the cypress key into an environment variable? I think it belongs there rather than checking it into the codebase.

`CYPRESS_ARGS="--record --key xxxx"`

Then

`"test_headless": "CYPRESS_baseUrl=${CYPRESS_baseUrl:=http://localhost:3000/} cypress run --browser electron ${CYPRESS_ARGS}"`

Probably doesn't make sense to upload recordings of running cypress tests from local dev environments each time.